packages/wdio-types/src/index.ts

```diff
@@ -21,6 +21,7 @@ declare global {
         interface MochaOpts { [key: string]: any }
         interface JasmineOpts { [key: string]: any }
         interface CucumberOpts { [key: string]: any }
+        interface TsNodeOpts { [key: string]: any }
```

Comment:
We can move this into the options.ts file and import it in the files where we need it directly. These options are only defined here so we can overwrite them in case someone has `@wdio/cucumber-framework` etc installed where we have access to these types.
```suggestion
```

Reply:
Great suggestion, that is good to know 👍